pkg/logql/parser_test.go

```diff
@@ -20,6 +20,25 @@ func TestParse(t *testing.T) {
 		exp Expr
 		err error
 	}{
+		{
+			// raw string
+			in: "count_over_time({foo=~`bar\\w+`}[12h] |~ `error\\`)",
```

Comment:
should this be:

```"count_over_time({foo=~`bar\w+`}[12h] |~ `error\`)",```

With only a single backlash because it's inside the new backticks?

Reply:
Yes, unless you have an alternative we can't escape backtick here, basically `foo |~ `error`` is interpreted as "foo |~" + identifier(error) + empty raw string . So I need to use " quoted query. and that requires escape. I call this go quote inception. Not sure I explained this correctly.

Reply:
yeah, ok, i was more just asking if this was a mistake or if it was a requirement of how to get the value in the test... no worries LGTM